### ISSUE TITLE
I want to contribute

### ISSUE THREAD

**Issue by yasharrashedi** [ajamous/DOTS#1]
Hi, what's the state of project? is it working now? are subscription and broadcasting of subscribers implemented? I can add even more functionalities (VAS) to the project if you're interested and work on a custom token for the project.

**Comment by ajamous** [comment]
@yasharrashedi Thanks for reaching out. Thank you very much for reaching out and sorry for the delay in responding. The project has not yet been started yet. I will keep in touch.

**Comment by ajamous** [comment]
@yasharrashedi am conducting a heavy research on how we can save a lot of time if we use Blockchained CPaaS APIs to achieve this project goal.

**Comment by yasharrashedi** [comment]
Hi, Sorry to say that but CPaaS may not be a good solution, because these services are highly centralized, we came across this solution: http://www.kamailio.org/dokuwiki/doku.php/development:proposal:distributed-message-queue but still have small issues and trying to solve them

**Comment by ajamous** [comment]
That’s true. CPaaS is currently centralized and part of my research to see the pathways to decentralizing it. The Kamailio distributed design is very interesting to me and is originally planned to be used for an important role of this project. What problems are you facing with it at this moment ? Can you share more details on limitations etc...

**Comment by yasharrashedi** [comment]
the biggest challenge is how much resources is used on host machines (to determine how much this node has to get paid?)

**Comment by ajamous** [comment]
@yasharrashedi Thanks for your feedback, as mentioned in the DOTS paper, the plan was to pay host machines based on transactions (billed minutes) through their network, the bigger machine resources are the highest calls load it can take therefore more billed minutes and rewards will be issued to them. Let me know what you think about this approach ?

**Comment by yasharrashedi** [comment]
@ajamous this is also hard to achive, but lot easier than counting resource usage
